### DISCUSSION TITLE
How to move y-axis to right hand side?

### DISCUSSION THREAD

**Discussion by somguyth** [highfestiva/finplot#180] [Q&A]
It's common to display the price on the right-hand-side of the chart (e.g. on TradingView and elsewhere).

I'm wondering if there's a way we can achieve this in finplot?

I can see that this is possible in pyqtgraph[1] but it's proving tricky for me to achieve that within finplot so far

[1] https://stackoverflow.com/questions/45800490/pyqt5-gui-with-pyqtgraph-plot-display-y-axis-on-the-right

**Comment by highfestiva** [comment]
You can try flipping all the 'left' to 'right' in `finplot/__init__.py`, but I always had some problems with that, but can't remember exactly what. If you make it work, please send me a pull request! :)

**Reply by highfestiva** [reply]
finplot now by default has the price axis to the right.
